README.md

```diff
@@ -55,9 +59,9 @@ For the list of supported signing algorithms, see [Creating and using an X.509 c
 
 The manifest JSON string defines the C2PA manifest to add to the file.
 
-A sample JSON manifest is provided in tests/fixtures/training.json.
+A sample JSON manifest is provided in [tests/fixtures/training.json](https://github.com/contentauth/c2pa-c/blob/main/tests/fixtures/training.json).
 
-```c++
```

Comment:
I found the syntax highlighting when previewing in visual studio worked better with C++ but if cpp is preferred and works for the public docs, i'm fine with that